src/hammer-vlsi/test.py

```diff
@@ -276,8 +276,11 @@ def test_yaml_tech_file(self) -> None:
         tech_yaml_filename = tech_dir + "/dummy28/dummy28.tech.yaml"
         with open(tech_yaml_filename, "w") as f:
             f.write(tech_yaml)
-        tech = hammer_tech.HammerTechnology.load_from_dir("dummy28", [tech_dir])
-        self.assertTrue(tech is not None, "Unable to load technology")
+        tech_opt = hammer_tech.HammerTechnology.load_from_dir("dummy28", [tech_dir])
+        if tech_opt is None:
+            self.assertTrue(tech_opt is not None, "Unable to load technology")
```

Comment:
Seems like this became an impossible assertion...?

Reply:
Oops. I had an refactor for the type checker that made that useless.

Reply:
It's actually an assertion that always fires which is correct(the test should fail if we can't load the technology).
Its a bit unclear in this case so I'm changing it.